docs/content/add-plugins/minio.md

```diff
@@ -18,11 +18,15 @@ You can follow [MinIO’s guide to install it](https://docs.min.io/minio/baremet
 
 After installing it, make sure MinIO is always running when your Medusa server is running. It’s recommended that you set up an alias to quickly start the MinIO server as instructed at the end of the installation guides in MinIO.
 
-:::warning
+### Change MinIO port
 
 In MinIO’s documentation, port `9000` is used for the address of the MinIO server. However, this collides with the port for the Medusa server. You must change the port for MinIO to another one (for example, port `9001`).
 
-:::
+After setting up and installing MinIO on your system/sub-system, you can run the following command to change MinIO port to `9001` (or any other available port) instead of `9000` to avoid the port clash
```

Comment:
```suggestion
After setting up and installing MinIO on your system/sub-system, you can run the following command to change MinIO port to `9001` (or any other available port) instead of `9000` to avoid the port clash:
```